docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -1,105 +1,192 @@
-# Host Risk Score- What is it?
+# Host Risk Score
 
-The Host Risk Score package consists of all the artifacts required to stand up the host risk scoring framework in your environment. This framework leverages transforms and visualizations in Kibana to identify the most suspicious hosts in your environment, based on alert activity on the hosts. 
-To deploy this framework in your environment, follow the steps outlined below.
+Host Risk Score is an experimental machine learning feature which assigns risk scores to hosts in a given Kibana space. Risk scores are calculated for each host by transforms on the alerting indices, and they update every hour with the most recent alerting activity. The Host Risk Score [package](https://github.com/elastic/detection-rules/releases) contains all of the required artifacts for setup. Host Risk Score feature is equipped with drilldown dashboards and additional Kibana features such as _the Host Risk Score Card on the Overview Page_ and _the Host Risk Keyword on the Alert Summary Flyout_ for an enhanced experience.
 
-# Detailed steps
+### Notes
```

Comment:
Do we want to move this to the bottom of the page?

Reply:
I think it's nice to have a disclaimer at the top, the details are in the bottom so it doesn't clutter the introduction too much